audio/engine.asm

```diff
@@ -2101,10 +2134,13 @@ Music_NewSong:
 	pop bc
 	ret
 
-GetMusicByte:
 ; returns byte from current address in a
 ; advances to next byte in music data
-; input: bc = start of current channel
+; input: 
+	; bc = start of current channel
+; output: 
+	; a = music byte
```

Comment:
Once again, cut down the verbosity. The function comment already said that the function returned in `a`, so there's no point in saying that twice.